index.html

```diff
@@ -6,6 +6,392 @@
 	<link rel="stylesheet" type="text/css" href="index.css">
 </head>
 <body>
-	
+
+	<!-- intro -->
+	<header class="row">
+
+		<img src="assets/yellow-pill.svg" alt="yellow pill">
+		<img src="assets/more-pills.svg" alt="a blue pill, a purple pill, and a yellow pill">
+		
+		<span class="subTitle">A Viget Exploration</span>
+		<h1>
+			<img src="assets/title.svg" alt="Improving Medical Adherence">	
+		</h1>
+		
+		<img src="assets/pills.svg" alt="a blue pill and yellow pill">
+		<img src="assets/purple-pills.svg" alt="two purple pills">
+
+		<!-- yellow-pill.svg is top left -->
+		<!-- more-pills.svg is top right -->
+		<!-- pills.svg is bottom left -->
+		<!-- purple-pills.svg is bottom right -->
+
+		<section class="whiteBox">
+			<div>
+				<h2>Patients who don't take medication as prescribed are at risk of worsening health</h2>
+				<p>Just in the U.S., non-adherence has been attributed to over $100 billion dollars in avoidable healthcare costs.<span>1</span> In developed countries, among patients with chronic illness, approximately 50% do not take medications as prescribed.<span>2</span></p>
+				<p id="whiteBoxQuote">“Increasing the effectiveness of adherence interventions might have a greater impact on the health of the population than any improvement in specific medical treatment.”</p>
+				<p id="WHO">World Health Organization, 2005 <span>3</span></p>
+			</div>
+		</section>
+
+		<!-- I decided to put all the pill bottles into one figure so I could combine them visually with CSS -->
+		<figure id="pillboxes">
+			<figure id="pillb1">
+				<img src="assets/pill-bottle3.svg" alt="red, rectangular pill bottle">
+			</figure>
+			<figure id="pillb2">
+				<img src="assets/pill-bottle.svg" alt="orange, fat pill bottle">
+			</figure>
+			<figure id="pillb3">
+				<img src="assets/pill-bottle2.svg" alt="orange, skinny pill bottle">
+			</figure>
+		</figure>
+	</header>
+	<!-- end intro -->
+
+	<!-- each section past the top header to be identified and provided appropriate whitespace within the class row -->
+
+	<!-- Why people don't adhere -->
+	<section class="row">
+		<span class="subTitle">WHY PEOPLE DON'T ADHERE</span>
+		<h2 class="sectionDescription">A myriad of factors conspire to prevent people from sticking to their treatment programs. But a few are most telling.</h2>
+		<section id="humanBody">
+
+			<!-- The diagram is a series of figures, with respective captions extending into tag bubbles -->
+
+			<figure>
+				<img src="assets/body.svg" alt="a gray outline of a human body">
+				<figure class="tagRight" id="comprehension">
+					<img src="assets/icon-comprehension.svg" alt="Two intertwined gears">
+					<figcaption>
+					<h3>Comprehension</h3>
+					“All of that medical jargon doesn’t make much sense to me but I don’t want to bother the doctor with my silly questions.”
+					</figcaption>
+				</figure>
```

Comment:
These seem odd as `<figure>`(s) too.  The main content here is the text, not the image.  The image is just a visual flourish to the main content -- the text.

I'd make these `<article>`(s), make the `<img>`(s) `alt=""` and get rid of that `<figcaption>` too, possibly converting it to just a `<div>` if you need a wrapper for styling.

*Note:* an HTML `<article>` is an article like an article of clothing, not like a news article.  It's nice for little things like these, which are too small to really be another `<section>`